admin-console/src/main/webapp/src/components/SystemBehaviorGroups/CreateEditBehaviorGroupModal.tsx

```diff
@@ -0,0 +1,109 @@
+    ];
+
+    const handleChange = (value: string, event: React.FormEvent<HTMLInputElement> | React.FormEvent<HTMLTextAreaElement>) => {
+        const target = event.target as HTMLInputElement;
+        setSystemBehaviorGroup(prev => ({ ...prev, [target.name]: target.value }));
+    };
+
+    const onSubmitLocal = React.useCallback(() => {
+        props.onSubmit(systemBehaviorGroup);
+    }, [ systemBehaviorGroup, props ]);
+
+    React.useEffect(() => {
+        setSystemBehaviorGroup(props.initialSystemBehaviorGroup ?? {});
+    }, [ props.initialSystemBehaviorGroup ]);
+
+    return (
+        <React.Fragment>
+            <Modal
+                variant={ ModalVariant.medium }
+                title={ `${ props.isEdit ? 'Update' : 'Create'} your System Behavior Group` }
+                isOpen={ props.showModal }
+                onClose={ props.onClose }
+            ><Form isHorizontal>
+                    <FormGroup label='Group Name' fieldId='displayName' isRequired
+                        helperText={ props.isEdit ? <HelperText><HelperTextItem variant="warning" hasIcon>
+                                                    If this field is modified it may affect exisiting behavior.
+                        </HelperTextItem></HelperText> : 'Enter a name for your group' }>
+                        <TextInput
+                            type='text'
+                            value={ systemBehaviorGroup.displayName }
+                            onChange={ handleChange }
+                            id='displayName'
+                            name="displayName"
+                        /></FormGroup>
+                    <FormGroup label='Action' fieldId='actions' isRequired>
+                        <Select
+                            variant={ SelectVariant.single }
+                            aria-label="Select action"
+                            placeholderText="Select an action"
+                            onToggle={ toggle }
+                            isOpen={ isOpen }
+                            onSelect={ onSelect }
+                            selections={ selected }
+                            menuAppendTo={ document.body }>
+                            { actionOption }
+                        </Select>
+                    </FormGroup>
+                    <FormGroup label='Recipient' fieldId='actions' isRequired>
+                        <Select
+                            variant={ SelectVariant.single }
+                            aria-label="Select recipiect"
+                            placeholderText="Select a recipient"
+                            onToggle={ toggle }
+                            isOpen={ isOpen }
+                            selections={ selected }
+                            onSelect={ onSelect }
+                            menuAppendTo={ document.body }>
+                            { recipientOption }
+                        </Select>
+                    </FormGroup>
```

Comment:
We are not going to have a lot of available actions/recipients in here; maybe we can simplify this to a single Select ? 
`Send email to admin`, `Send email to all users`. In the future we might have some other options.

Reply:
For sure.